data/reusables/repositories/section-links.md

```diff
@@ -1,23 +1,22 @@
-You can link directly to a section in a rendered file by hovering over the section heading to expose {% octicon "link" aria-label="the link" %}.
+You can link directly to any section that has a heading. To view the automatically generated anchor in a rendered file, hover over the section heading to expose the {% octicon "link" aria-label="the link" %} icon then click the icon to display the anchor in your browser.
 
 ![Screenshot of a README for a repository. To the left of a section heading, a link icon is outlined in dark orange.](/assets/images/help/repository/readme-links.png)
 
-[Headings](/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings) automatically create anchors which have default names created from the text of the heading, with the following rules:
+If you need to determine the anchor for a heading in a file you are editing, you can use the following rules:
 
  * Generated identifiers are UTF-8 URI fragments.
     * See [RFC 3986: Uniform Resource Identifier (URI): Generic Syntax, Section 3.5](https://www.rfc-editor.org/rfc/rfc3986#section-3.5).
  * All ASCII letters are converted to lower-case.
  * Spaces (UTF-8 codepoint `0x20`) are replaced by hyphens (`-`).
  * Leading and trailing whitespace is removed.
  * All other whitespace characters are removed.
- * Paired formatting markup characters are removed, leaving only their contents (e.g., `_italics_` becomes `italics`).
+ * Paired formatting markup characters are removed, leaving only their contents (for example, `_italics_` becomes `italics`).
  * Punctuation and other characters not allowed in a URI fragment are removed.
  * Other multi-byte UTF-8 characters which are valid in URI fragments are not modified.
- * Percent Encoding is not performed.
```

Comment:
I only have issues with these — I seem to have been getting multibytes always percent encoded, so also the "are not modified" above it doesn't sound correct to me in all the cases.

(TBC, need to do a few wiki experiments to confirm exactly…)

I'm afraid for accented characters, a general note that the user should check the generated fragment first before linking to it might be better.

Reply:
EDIT: Submitting fresh review instead of outdated content.